webapi_tests/orientation/__init__.py

```diff
@@ -2,5 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from webapi_tests.orientation.device_motion_test import DeviceMotionTestCommon
+from webapi_tests.orientation.test_device_motion import TestDeviceMotion
```

Comment:
I prefer to put them in the same suite since it's all about the orientation and moving. They also share the same script.